### PR TITLE
Electron and node upgrade

### PR DESCRIPTION
**Pull request by nucleardreamer**
- Upgrading electron to `v3.0.11`
- Upgrading node to `v10.14.2`
- Updating `package-lock.json` for minor versions

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ services:
   oak:
     build: .
     container_name: oak
-    image: oaklabs/oak:4.3.4
+    image: oaklabs/oak:5.0.0
```
Comment (rjoffray) [inline docker-compose.yml]: this should match the 5.0.0-rc3 in the deploy shell script

File: docs/start-building.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ First, create a new directory where all the files would live. In this directory
       "description": "App repository as an example",
       "main": "src/index.js",
       "engines": {
-        "node": "8.11.3"
+        "node": "10.2.1"
```
Comment (rjoffray) [inline docs/start-building.md]: this should be v10.14.2

File: manifest.yml [hunk 3]
```diff
@@ -1,14 +1,14 @@
-image: oaklabs/oak:4.3.4
+image: oaklabs/oak:5.0.0
```
Comment (rjoffray) [inline manifest.yml]: should these also match the 5.0.0-rc3 or will this matter?

File: readme.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # Oak
 
-[![release](https://img.shields.io/badge/release-v4.3.4-green.svg)](https://github.com/OakLabsInc/oak/releases/tag/4.3.4)
-[![node](https://img.shields.io/badge/node-v8.11.3-green.svg)](https://github.com/nodejs/node/releases/tag/v8.11.3)
-[![electron](https://img.shields.io/badge/electron-v1.8.8-green.svg)](https://github.com/electron/electron/releases/tag/v1.8.8)
+[![release](https://img.shields.io/badge/release-v5.0.0-green.svg)](https://github.com/OakLabsInc/oak/releases/tag/5.0.0)
```
Comment (rjoffray) [inline readme.md]: this is just the docs but maybe should also match 5.0.0-rc3

File: .dockerignore [hunk 5]
```diff
@@ -13,3 +13,4 @@ docker-deploy.sh
 readme.md
 .cache/
 build/
+_config.yml
```
Comment (rjoffray) [inline .dockerignore]: add a newline at the end of the file

File: .gitignore [hunk 6]
```diff
@@ -7,3 +7,4 @@ node_modules
 .vscode/
 build/release/
 .cache/
+_config.yml
```
Comment (rjoffray) [inline .gitignore]: add a newline at the end of the file

File: examples/inject-scripts/.nvmrc [hunk 7]
```diff
@@ -1 +1 @@
-v10.2.1
```
Comment (rjoffray) [inline examples/inject-scripts/.nvmrc]: should this be v10.14.2

File: examples/multiple-windows/.nvmrc [hunk 8]
```diff
@@ -1 +1 @@
-v10.2.1
```
Comment (rjoffray) [inline examples/multiple-windows/.nvmrc]: should this be v10.14.2

File: examples/simple-script/.nvmrc [hunk 9]
```diff
@@ -1 +1 @@
-v10.2.1
```
Comment (rjoffray) [inline examples/simple-script/.nvmrc]: should this be v10.14.2

File: examples/simple-url/.nvmrc [hunk 10]
```diff
@@ -1 +1 @@
-v10.2.1
```
Comment (rjoffray) [inline examples/simple-url/.nvmrc]: should this be v10.14.2

**Review by rjoffray — DISMISSED** [review]

**Comment by nucleardreamer** [comment]
Closes #35

**Pull request merged**